lib/mock/index.js

```diff
@@ -292,6 +292,7 @@ function sendFile (req, res, next, file) {
         return name.toLowerCase() === "content-disposition";
       });
 
+      res.set("Content-Type", file.mimetype);
```

Comment:
files placed on disk are not `/somedir/random-characters`; I think they used to be `/somedir/random-character.jpg` for example. The missing extension is throwing off the auto-mimetype that sendfile (I think provides) so this is a notable change to callout to folks.

I didn't think it was worth somehow renaming the file on disk to keep this behavior.

Reply:
Can you add a 4.0.0 entry to [the changelog](https://github.com/APIDevTools/swagger-express-middleware/blob/master/CHANGELOG.md) and call-out any major changes like this?